src/pages/tools/MonsterSlayerPage.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect } from 'react';
+import {
+    TypedUseSelectorHook,
+    useDispatch,
+    useSelector as useReduxSelector
+} from 'react-redux';
+import {
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow
+  } from '@material-ui/core';
+
+import { loadStageScores } from '../../store/stageScoreActions';
+import { StageScore } from '../../domain/stageScore';
+import { AppState } from '../../store/rootReducer';
+
+export const useSelector: TypedUseSelectorHook<AppState> = useReduxSelector;
+
+export const MonsterSlayerPage: React.FC<{}> = () => {
+    const stageScores: StageScore[] = useSelector(
+        state => state.stageScoreState.stageScores
+    );
+
+    const dispatch = useDispatch();
+
+    useEffect(() => {
+        dispatch(loadStageScores());
+    }, [dispatch]);
+
+    if (stageScores.length == 0) {
+        return (
+            <Paper><h1>Monster Slayer</h1><h2>No current Monster Slayer event</h2></Paper>
```

Comment:
I would prefer to keep this html spaced even though it's shorter just for consistency and minimal line changes when updating